ときどきCIのテストが落ち、ReRunすると落ちない

## 💡 Summary

ときどきテストが失敗する。
再現性が低い、おしまいだ。

## 📝 再現方法

1. テストCIをまわす
2. 時々落ちる

＞わからない！！！＜

## 🤬 失敗例

https://github.com/momocus/sakazuki/actions/runs/9731029983/job/26854920162

エラー内容は、酒フォームで酒の名前から加水・原酒を原酒に自動補完する機能。
原酒の補完ができてない。

## エラーログ

```console
...........................................................................................................................................................................................................................F................................................................................................

Failures:

  1) Sake Form Completion for Detail for detail accordion warimizu is completed 原酒
     Failure/Error: expect(page).to have_select("sake_warimizu", selected:)
       expected to find visible select box "sake_warimizu" that is not disabled but there were no matches. Also found "", which matched the selector but not all filters. 

     [Screenshot Image]: /home/runner/work/sakazuki/sakazuki/tmp/capybara/failures_r_spec_example_groups_sake_form_completion_for_detail_for_detail_accordion_warimizu_is_completed_原酒_309.png


     # ./spec/system/sake_form_completion_for_detail_spec.rb:178:in `block (4 levels) in <top (required)>'

Finished in 1 minute 48.9 seconds (files took 3.81 seconds to load)
3[16](https://github.com/momocus/sakazuki/actions/runs/9731029983/job/26854920162#step:9:17) examples, 1 failure

Failed examples:

rspec ./spec/system/sake_form_completion_for_detail_spec.rb:[17](https://github.com/momocus/sakazuki/actions/runs/9731029983/job/26854920162#step:9:18)5 # Sake Form Completion for Detail for detail accordion warimizu is completed 原酒

Stopped processing SimpleCov as a previous error not related to SimpleCov has been detected
Coverage report generated for RSpec to /home/runner/work/sakazuki/sakazuki/coverage/coverage.xml. 310 / 321 LOC (96.57%) covered
Error: Process completed with exit code 1.
```

--- a/spec/support/capybara.rb
+++ b/spec/support/capybara.rb
@@ -23,6 +23,10 @@ Capybara.configure do |config|
 
   # "data-testid"をCapybaraのclick_linkなどで使えるように、Optional attributeに登録する
   config.test_id = "data-testid"
+
+  # Capybaraのアサーションが失敗したときに自動再試行する時間
+  # JS処理が間に合わないなどフレーキーなテストへの対応するため少し長くする
+  config.default_max_wait_time = 5
 end
 
 RSpec.configure do |config|

